packages/terra-arrange/package.json

```diff
@@ -31,8 +31,15 @@
     "release:major": "npm test && node ../../scripts/release/release.js major",
     "release:minor": "npm test && node ../../scripts/release/release.js minor",
     "release:patch": "npm test && node ../../scripts/release/release.js patch",
-    "test": "npm run test:spec",
-    "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json"
+    "test": "npm run test:spec && npm run test:nightwatch-default",
+    "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
+    "test:all": "npm run test:nightwatch-default && npm run test:nightwatch-chrome && npm run test:nightwatch-firefox && npm run test:nightwatch-safari",
+    "test:nightwatch-default": "SPECTRE_TEST_SUITE=terra-button node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js",
```

Comment:
Ah good catch, I missed that. Updated: https://github.com/cerner/terra-core/pull/220/commits/40e7a9db165be4a25b3414381d07742828c9f2ea